trie/patriciaMerkleTrie.go

```diff
@@ -269,6 +269,17 @@ func (tr *patriciaMerkleTrie) Commit() error {
 	return nil
 }
 
+// GetStorageStats will returns trie storage operations statistics as string
+func (tr *patriciaMerkleTrie) GetStorageStats() string {
+	dbWithStats, ok := tr.trieStorage.(storageManagerWithStats)
```

Comment:
will this be called many times and fail? there will be a lot of warns in the log. 
could all StorageManagers have a GetStatsCollector() that returns "" on ToString()? 
if not, could you also add a unit test for the branch !ok?